cluster/sync.sh

```diff
@@ -101,5 +118,24 @@ function wait_ready() {
     done
 }
 
+function wait_ready_operator() {
+    # Wait until the operator becomes consistently ready on all nodes
+    for dep in nmstate-operator; do
+        # We have to re-check desired number, sometimes takes some time to be filled in
+        if ! eventually isOperatorOk $dep; then
+            echo "Deployment $dep haven't turned ready within the given timeout"
+            exit 1
+        fi
+
+        # We have to re-check desired number, sometimes takes some time to be filled in
+        if ! consistently isOperatorOk $dep; then
+            echo "Deployment $dep is not consistently ready within the given timeout"
+            exit 1
+        fi
+    done
+}
+
+deploy_operator
+wait_ready_operator
 deploy
```

Comment:
This is suppose to be done by operator we have to remove it (at the end is what we are testing).